### DISCUSSION TITLE
What are my next steps after the course?

### DISCUSSION THREAD

**Discussion by YasmeenAmr** [HossamAbubakr/Udacity-Full-Stack-Classroom#6] [Q&A] [answered]
lend me a hand with this what should I do after graduation?

**Comment by HossamAbubakr** [comment, marked as answer]
Hello Yasmeen,

The next step would be to start picking a frontend framework/library of your choosing and getting into the Frontend of things. I know you are already done with your React course so that's great!

So now it's time to combine your knowledge into a full-stack application.

Preferably a PERN stack one as it's what you should be familiar with right now.

I suggest starting with something like [this](https://www.interviewbit.com/blog/full-stack-projects-with-source-code/). It includes a list full of both ideas and reference source codes.

And once you build a couple of apps I suggest exploring some technologies outside your stack such as [GraphQL](https://www.youtube.com/watch?v=eIQh02xuVw4)

In the meantime, I suggest taking a look at these free scholarships which will start soon.
[ALX-T](https://www.alx-t.com/)
[Bertelsmann](https://www.udacity.com/bertelsmann-tech-scholarships)
